src/styles/blocks/carousel.scss

```diff
@@ -0,0 +1,110 @@
+.carousel {
+  display: flex;
+  gap: 20px;
+  overflow-x: scroll;
+  scroll-behavior: smooth;
+  
+  &::-webkit-scrollbar {
+    background-color: transparent;
+    height: 0;
+  }
+
+  &__card {
+    box-sizing: border-box;
+    display: block;
+    margin: 0 -20px 25px;
+    transition: transform 300ms;
```

Comment:
Consider using 'padding' instead of 'margin' to avoid negative values and prevent overlapping issues.